flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/main/java/com/ververica/cdc/connectors/mysql/source/MySqlDataSourceOptions.java

```diff
@@ -230,4 +230,12 @@ public class MySqlDataSourceOptions {
                     .defaultValue(true)
                     .withDescription(
                             "Whether send schema change events, by default is true. If set to false, the schema changes will not be sent.");
+
+    @Experimental
+    public static final ConfigOption<Boolean> ENABLE_COLUMN_COMMENTS =
+            ConfigOptions.key("enable.column.comments")
```

Comment:
Would it be better to use "column-comments.enabled"?

Reply:
ok, i fix it.